std/format/spec.d

```diff
@@ -902,22 +902,13 @@ FormatSpec!Char singleSpec(Char)(Char[] fmt)
     assertThrown!FormatException(singleSpec("%%"));
 }
 
+// @@@DEPRECATED_[2.107.0]@@@
+deprecated("enforceValidFormatSpec was accidentally made public and will be removed in 2.107.0")
 void enforceValidFormatSpec(T, Char)(scope const ref FormatSpec!Char f)
```

Comment:
Maybe replace the entire function with a deprecated public alias?

Reply:
The deprecation message would then talk about an alias instead of a function. Don't know if this makes a difference...